### DISCUSSION TITLE
Feature collection

### DISCUSSION THREAD

**Discussion by escamoteur** [DevsHelpDevs/DevsHelpDevs#6] [Ideas]
Please help me collect features that you think we will need for such a platform.

**Comment by escamoteur** [comment]
I just added a preliminary user specification. This will need to be broken down to issues in the future but should give an initial idea of the planned system. 

https://github.com/DevsHelpDevs/DevsHelpDevs/blob/master/user-specification.md

**Reply by escamoteur** [reply]
Please add your comments on it here so that it can evolve into a final document that serves as a base of a common understanding what this platform is all about.

**Comment by Jhanbury** [comment]
One feature I think might be good would be the support for Social/Open Source Links like the following:

- Github Account/Repository
- Portfolio Site
- Reference Project etc
- Blogs
- Youtube/Twitch Accounts

I feel this has two benefits for applicants:

1. Reference Points for Story
    - It might be difficult for a candidate to express their problem via text alone
    - Having a reference would add clarity as donators can see the applicants work

2. Secondary benefit for applicants
    - The site may not be able to guarantee that an applicant will get a donation
    - This feature may offer some secondary benefits for applicants:
        - Exposure
            - Blogs, Projects, Portfolio sites, Video platforms
        - New connections
            - Github, portfolio sites

**I see this as an optional add-on to an application or applicant and not a required field**

Would love to hear what people think about this?

**Reply by escamoteur** [reply]
That's an excellent idea, we should add this to out upcoming data model

**Reply by Evavic44** [reply]
> One feature I think might be good would be the support for Social/Open Source Links like the following:
> 
> * Github Account/Repository
> * Portfolio Site
> * Reference Project etc
> * Blogs
> * Youtube/Twitch Accounts
> 
> I feel this has two benefits for applicants:
> 
> 1. Reference Points for Story
>    
>    * It might be difficult for a candidate to express their problem via text alone
>    * Having a reference would add clarity as donators can see the applicants work
> 2. Secondary benefit for applicants
>    
>    * The site may not be able to guarantee that an applicant will get a donation
>    * This feature may offer some secondary benefits for applicants:
>      
>      * Exposure
>        
>        * Blogs, Projects, Portfolio sites, Video platforms
>      * New connections
>        
>        * Github, portfolio sites
> 
> **I see this as an optional add-on to an application or applicant and not a required field**
> 
> Would love to hear what people think about this?

Great idea.

**Comment by seunbayo** [comment]
I think we should consider using https://linear.app/ to break this thing down then we can further integrate with GitHub and slack.

I can set it up if we are ready because I do use it for other projects

**Reply by escamoteur** [reply]
We can use Notion for that

**Reply by seunbayo** [reply]
When we start creating and solving tickets, does notion have a Slack integration?

Have you check out linear.

**Reply by escamoteur** [reply]
yep it has, check out we have a #notion channel

**Comment by Evavic44** [comment]
I think we should be actively involved in Open Source on GitHub for more awareness and setup GitHub Sponsor so people can donate.
> Solutions
- Add members to devshelpdevs organization on GitHub (I noticed there are no members added to this Organization)
- Setup GitHub sponsor, so people can donate to the organization.
- Creating and working on OpenSource projects as a team to make the community active and attract new members and sponsors.

**Reply by escamoteur** [reply]
Sure we can add people to the org, no problem.
githubsponsors for orgs is a bit tricky. what would be the difference to the current moneypool?

What do you mean by working on Open source projects? DevsHelpDevs is an open source project itself

**Reply by Evavic44** [reply]
> Sure we can add people to the org, no problem.
> githubsponsors for orgs is a bit tricky. what would be the difference to the current moneypool?
> 
> What do you mean by working on Open source projects? DevsHelpDevs is an open source project itself

- I think it will make a difference, even if it's not much at the start, if we're serious about it, It'll definitely grow in time.
- I meant creating one, like an app or website done by community members.
